src/main/java/com/exasol/adapter/sql/SqlFunctionAggregate.java

```diff
@@ -42,23 +42,23 @@ public SqlFunctionAggregate(AggregateFunction function, List<SqlNode> arguments,
     public AggregateFunction getFunction() {
         return function;
     }
-    
+
     public String getFunctionName() {
         return function.name();
     }
 
     public boolean hasDistinct() {
         return distinct;
     }
-    
+
     @Override
     public String toSimpleSql() {
         List<String> argumentsSql = new ArrayList<>();
         for (SqlNode node : arguments) {
             argumentsSql.add(node.toSimpleSql());
         }
-        if (argumentsSql.size() == 0) {
-            assert(getFunctionName().equalsIgnoreCase("count"));
+        if (argumentsSql.isEmpty()) {
+            assert (getFunctionName().equalsIgnoreCase("count"));
```

Comment:
Please remove surrounding parentheses. Assert is not like a regular method call.

Reply:
fixed